orttraining/orttraining/core/optimizer/graph_transformer_utils.cc

```diff
@@ -77,6 +78,7 @@ std::vector<std::unique_ptr<GraphTransformer>> GeneratePreTrainingTransformers(
       rule_transformer->Register(make_unique<CastElimination>());
       rule_transformer->Register(make_unique<DivMulFusion>());
       rule_transformer->Register(make_unique<EliminateDropout>());
+      rule_transformer->Register(make_unique<GemmTransposeFusion>());
```

Comment:
missing registry in the file below, thus transformation didn't kick in.
onnxruntime/core/optimizer/graph_transformer_utils.cc

Reply:
good catch, thanks!